### PR TITLE
Stripe Windows Ephemeral Disks on Boot

### PR DESCRIPTION
**Pull request by Guslington**
add to user data to run on boot
```
    Property('UserData', FnBase64(FnJoin("",[
      "<powershell>\n",
      "C:/base2/bin/Stripe-Windows-Ephemeral-Disks.ps1\n",
      "</powershell>\n",
    ])))
```
Create scheduled task to run on boot if restarting the server is required

### CODE REVIEW

File: files/default/windows/Stripe-Windows-Ephemeral-Disks.ps1 [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+#################################################
+#  Given a device (e.g. xvda), strip off "xvd" and convert the remainder to the appripriate SCSI ID
+#################################################
+function GetSCSI {
+	Param([string]$device)
+
+	$deviceSuffix = $device.substring(3)      # remove xvd prefix
+
+  if ($deviceSuffix.length -eq 1) {
```
Comment (toshke) [inline files/default/windows/Stripe-Windows-Ephemeral-Disks.ps1]: looks like spacing issue, not sure if affects PowerShell execution

**Comment by aaronwalker** [comment]
retest this please

**Comment by aaronwalker** [comment]
retest this please

**Comment by aaronwalker** [comment]
ok to merge

**Pull request merged**